tool/src/java/org/sakaiproject/profilewow/tool/producers/SearchResultProducer.java

```diff
@@ -308,6 +309,34 @@ else if ((profile.getHidePublicInfo() != null) && (profile.getHidePublicInfo().b
 		return searchResults;
 	}
 
+	private boolean getSakai(SakaiPerson profile) {
+		  // User account types
+		  String TYPE_STUDENT = "student";
+		  String TYPE_STAFF = "staff";
+		  String TYPE_THIRDPARTY = "thirdparty";
+		  //String TYPE_OFFER = "offer";
+		  
+		log.debug("User: " + profile + "Type: " + profile.getTypeUuid());
+		User user = null;
+		try {
+			user = userDirectoryService.getUser(profile.getAgentUuid());
+			String type = user.getType();
+			log.debug("User: " + user.getDisplayId() + ", type: "+ user.getType());
```

Comment:
I think nowadays this works with placeholders, e.g.

			log.debug("User: {}, type: {}", profile, user.getType());